Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2976,7 +2976,7 @@ class DeferredCountable<T> {
     }
     
     func fill(_ value: T) {
-        await.fill(value)
+        await.fillIfUnfilled(value)
```

Comment:
This should either be included in a separate PR, since it solves a different case, or at the _very least_ a unique commit with its own message.

Reply:
I will make a separate commit